specifyweb/frontend/js_src/lib/components/Toolbar/WbsDialog.tsx

```diff
@@ -241,11 +241,7 @@ export function DataSetsDialog({
                           : undefined
                       }
                     >
-                      {dataset.uploadplan ? (
-                        <TableIcon label name={dataset.uploadplan.baseTableName} />
-                      ) : (
-                        <TableIcon label name="Workbench" />
-                      )}
+                      <TableIcon label name={dataset.uploadplan?.baseTableName ?? 'Workbench'} />
```

Comment:
now this looks good, but just a FYI for the future, that even without the fancy `?.` syntax, the original version you had could have been simplified by moving the `?` inside as much as possible so that the duplication is avoided

```tsx
<TableIcon label name={dataset.uploadplan ? dataset.uploadplan.baseTableName : 'Workbench'} />
```